test/distributed/org/apache/cassandra/distributed/test/accord/AccordLoadTest.java

```diff
@@ -172,4 +173,13 @@ protected Logger logger()
     {
         return logger;
     }
+
+    public static void main(String[] args) throws Throwable
```

Comment:
why do we need a main method?

Reply:
It allows me to avoid starting up junit which just plays nicer locally.